app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,6 @@
+<% provide(:title, 'Contact') %>
+<h1>Contact</h1>
+<p>
+  Contact the Ruby on Rails Tutorial about the sample app at the
```

Comment:
[IMO]
use I18n for this text `Contact the Ruby on Rails Tutorial about the sample app at the`